src/modules/mavlink/mavlink_main.cpp

```diff
@@ -1476,6 +1476,11 @@ Mavlink::update_radio_status(const radio_status_s &radio_status)
 			/* this indicates spare bandwidth, increase by 2.5% */
 			_radio_status_mult *= 1.025f;
 		}
+
+		/* Constrain radio status multiplier between 1% and 100% to allow recovery */
+		if (_radio_status_mult > 1.0f) { _radio_status_mult = 1.0f; }
```

Comment:
_radio_status_mult
```suggestion
		_radio_status_mult = math::constrain(_radio_status_mult, 0.01f, 1.0f);
```

Reply:
These suggestions seem to be a new feature.  I'd just click commit suggestion, but it looks like you missed deleting the first conditional